xcom2-launcher/xcom2-launcher/Classes/GlobalSettings.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Sentry;
+
+namespace XCOM2Launcher.Classes {
+    [JsonObject(MemberSerialization.OptIn)]
+    internal sealed class GlobalSettings {
+        private static readonly log4net.ILog Log = log4net.LogManager.GetLogger(nameof(GlobalSettings));
+
+        [JsonProperty] public Version Version;
```

Comment:
I think it makes sense to name this `MaxVersion` since it's not updated when an old version is opened

Reply:
Makes sense.